### PR TITLE
Fixing contact info Test

### PR DESCRIPTION
**Pull request by HerbertYiga**
Ticket Id
https://issues.openmrs.org/browse/RATEST-92

Description

This pr is deepened on by https://github.com/openmrs/openmrs-distro-referenceapplication/pull/333 so as to fix the failing tests  cc @k-joseph

### CODE REVIEW

**Comment by dkayiwa** [comment]
@HerbertYiga is there any reason why you do commit messages in capital letters?

**Comment by HerbertYiga** [comment]
> is there any reason why you do commit messages in capital letters?

absolutely no reason at all !,fixed this

File: src/main/java/org/openmrs/uitestframework/page/Page.java [hunk 1]
```diff
@@ -345,10 +345,11 @@ public Boolean containsText(String text) {
 
     public List<String> getValidationErrors() {
         List<String> validationErrors = new ArrayList<String>();
-        for (WebElement webElement : driver.findElements(By.className("error"))) {
```
Comment (kaweesi) [inline src/main/java/org/openmrs/uitestframework/page/Page.java:348]: i can see the field with class error, are u sure you are pointing to the right element, there are lots on field-error elements on the same page

Comment (HerbertYiga) [reply src/main/java/org/openmrs/uitestframework/page/Page.java:348]: > i can see the field with class error, are u sure you are pointing to the right element, there are lots on field-error elements on the same page

first class error never exists,secondly this driver.findElements(By.className("field-error") picks all the classess with (field-error)  and thats what we deal with in the for loop @k-joseph

Comment (kaweesi) [reply src/main/java/org/openmrs/uitestframework/page/Page.java:348]: have you tried checking, use your chrome tools on demo or qa instances and check, i can see one .error element and i wanna be sure we know difference between it and these field errors!

Comment (HerbertYiga) [reply src/main/java/org/openmrs/uitestframework/page/Page.java:348]: > have you tried checking, use your chrome tools on demo or qa instances and check, i can see one .error element and i wanna be sure we know difference between it and these field errors!

ok am gona test this and get back here

**Comment by kdaud** [comment]
@HerbertYiga are you still working on this PR ?

**Comment by HerbertYiga** [comment]
> re you still working on this PR

i will have to close this nolonger assigned to the ticket

**Pull request closed without merging**